core/trino-main/src/main/java/io/trino/execution/CreateMaterializedViewTask.java

```diff
@@ -61,8 +60,7 @@ public CreateMaterializedViewTask(
             AccessControl accessControl,
             SqlParser sqlParser,
             AnalyzerFactory analyzerFactory,
-            MaterializedViewPropertyManager materializedViewPropertyManager,
-            FeaturesConfig featuresConfig)
+            MaterializedViewPropertyManager materializedViewPropertyManager)
```

Comment:
Can you please move this change to a separate PR?

Reply:
see #12027